docker-php-source

```diff
@@ -36,6 +36,11 @@ case "$1" in
 		;;
 	delete)
 		rm -rf "$dir"
+        # To maintain BC we create this directory because many people
+        # were extracting some of their custom extensions in this directory
+        # See https://github.com/docker-library/php/issues/266
+        # Was breaked by https://github.com/docker-library/php/pull/256
+        mkdir -p /usr/src/php/ext
```

Comment:
This will break our detection of the php source being already extracted on line 22

Also, this is somehow spaces instead of tabs.